script/Ciel/Day 5/05_01_CIEL05_4D.txt

```diff
@@ -769,7 +769,7 @@ I still can't quite believe it but... she really defeated an Ancestor, right?
 [sha:85bb9e1a2e68d5536f3fc9a41b30670b4fe91d0d]{
 -- Page 842, Offset 13113.
 -- 「はあ？　バカいってんじゃねえぞ<七面鳥|ターキー>。
-"%{i}Whaaa%{/i}? Where the fuck did you come up with that idea, you turkey.
+"Whaaa? Where the fuck did you come up with that idea, you turkey?
```

Comment:
My man what is the rhyme and reason for these exclamation changes

Reply:
Sorry, be exclamation here I was referring to the `Whaa?`. The ? after turkey is fine.